README.md

```diff
@@ -35,45 +44,64 @@ Master branch is protected so we would work on develop branch for daily work and
 Merging to master branch would require one of us to approve the changes.
 Pull request needs to be created for reviewing new changes to existing codebase.
 
-
 ## Test
-### iOS
+
+### iOS Test
+
 We are incorporating Unit tests as our default workflow.
 The tests could be found in test branch and tests folder.
 
 ## Issues
 
-We would use JIRA, Git Issues or any Issue tracker.
+We are using GitHub Issues, GitHub Projects to track our issues, feature requests for every user story.
+Please file a Bug request or new Feature request [here](https://github.com/SensehacK/capstone-team4/issues)
 
 ## Timeline
-Schedule for deliverables.
+
+Schedule for deliverables is mentioned in our Milestones. Our Product owner and project manager would prioritize which tasks takes precedence and would assign them to their specific Milestones accordingly.
+You could find our milestones [here](https://github.com/SensehacK/capstone-team4/milestones)
 
 ## Environment
 
-- OS:
-	- Mac: Catalina 10.15
-	- Windows: Windows 10
+### OS
+
+- Mac: Catalina 10.15
+- Windows: Windows 10
+
+### IDE
+
+- Xcode 11.2.1
+- VS Code
+
+### Browsers
 
-- IDE: 
-	- Xcode 11.2.1
-	- VS Code
+- Firefox Nightly /Dev /Main
+- Safari /Technical Preview
+- Microsoft Edge Dev (Chromium)
 
+### Tools
 
-- Browsers:
-	- Firefox Nightly /Dev /Main
-	- Safari /Technical Preview
-	- Microsoft Edge Dev ( Chromium )
-	
-- Version Control : Git
-- Team management : JIRA, Github Project
-- Documents : Google docs
+- Version Control: Git
+- Team management: Github Project, JIRA
+- Group Communication: FaceTime, Zoom, WhatsApp Group
+- GTD App: Things, Trello
+- Documents: Microsoft Office Suite, Google Docs
+- Shared Drive: Google Drive
 - Terminal : iTerm
 - Writer: iA Writer
 - Terminal : iTerm
+- Image: Adobe PhotoShop
+- Icon Design: Adobe illustrator
+- Prototype: Sketch
 
 ## Shared Links
 
 ## Credits
 
 ### Team
 
+Abbas Shamshi(Front End Dev, Database Designer)
+Endri Zylali(Android Dev, Back End Dev)
+[Kautilya Save](https://www.linkedin.com/in/kautilyasave/)(iOS Dev, Architect)
+Manasi Tayade(Business Analyst, Designer)
+Rushikesh Nage(Project Manager, Scrum Master, Tester)
```

Comment:
https://www.linkedin.com/in/rushikesh-nage06/
Add this linkedin link at the name.

@ManasiTayade @endri969 @Abbas-shamshi do you guys want add your linkedin links to your name?

Reply:
I have taken the liberty of linking their names by myself. They can raise one more PR if they want to have changes. @rtrend96